_posts/en/newsletters/2021-03-03-newsletter.md

```diff
@@ -81,7 +81,23 @@ repo], [Hardware Wallet Interface (HWI)][hwi repo],
 [Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
 BOLTs][bolts repo].*
 
-- [Bitcoin Core #16546][] External signer support - Wallet Box edition FIXME:jnewbery
+- [Bitcoin Core #16546][] introduces a new signer interface, allowing Bitcoin
+  Core to interact with external hardware signing devices through the
+  [HWI][topic hwi] or any other application which uses the same arguments and
+  return values.
+
+    Bitcoin Core has been able to interface with hardware signers using HWI
+    [since Bitcoin Core version 0.18][hwi release]. Until this PR, however, [the
+    process][hwi old process] required use of the command line to copy and
+    paste data between Bitcoin Core and HWI. This PR allows Bitcoin Core to
+    communicate directly with HWI, simplifying the user experience. The PR includes
```

Comment:
Optional: This PR simplifies the user experience by enabling Bitcoin Core to directly communicate with HWI.

Reply:
done